PURL-TYPES.rst

```diff
@@ -87,6 +87,25 @@ bitnami
       pkg:bitnami/wordpress@6.2.0
       pkg:bitnami/wordpress@6.2.0?arch=arm64
 
+brew
+----
+``brew`` for Homebrew-based packages:
+
+- The default repository (tap) is ``homebrew/core``.
+
+  - The tap syntax is ``https://github.com/{org}/Homebrew-{tap}``, so
+    ``homebrew/core`` corresponds to the URL ``https://github.com/homebrew/homebrew-core``.
+- The ``name`` is the formula name.
+- The ``version`` is the formula version.
+- Qualifier ``tap_url``: for taps that are not on GitHub or otherwise require an explicit URL,
+  this is the full URL to the tap.
```

Comment:
**question (non-blocking):** Should there be another parameter to name the tap? We use the named tap at work, e.g.

    brew tap myco/brewhouse git@git.myco.com:brew/house.git

and the formula, uh, `myco-ctl` could have a purl

    pkg:brew/myco/brewhouse/myco-ctl?tap_url=git@git.myco.com:brew/house.git

and brew would know how to name the tap if it needed to be tapped, or perhaps

    pkg:brew/myco/brewhouse/myco-ctl?tap_url=git@git.myco.com:brew/house.git&tap_name=myco/brewhouse

or more simply

    pkg:brew/myco-ctl?tap_url=git@git.myco.com:brew/house.git&tap_name=myco/brewhouse

Reply:
My 0.02c would be for your first form:

```
pkg:brew/myco/brewhouse/myco-ctl?tap_url=git@git.myco.com:brew/house.git
```

I _think_ that's unambiguous: the `github.com/{org}/homebrew-{tap}` logic only applies when a `tap_url` isn't present, so I think this is the shortest form that conveys all needed information. But maybe I've overlooked something?